src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -95,6 +96,12 @@ void FrontEnd::translate_graph(const ov::frontend::InputModel::Ptr& model,
         auto input_shape = input_tensor_place->get_partial_shape();
         auto input_type = input_tensor_place->get_element_type();
 
+        // in case of cutting graph, types of custom inputs can be undefined,
+        // according to MO help, fp32 is used by default in such cases
+        if (input_type == element::undefined) {
+            input_type = element::f32;
```

Comment:
Couldn't find this in MO help, could you point me where it is stated? Also what will happen if `input_type` will be specified as dynamic? It should also work I think.

Reply:
Please see https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/utils/cli_parser.py#L492